staging/src/k8s.io/apiserver/pkg/authentication/request/union/union.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright 2014 The Kubernetes Authors.
```

Comment:
weird. Why the copyright statement is changed but there is not change in this file?

Reply:
I had some changes but was later removed.